.github/workflows/ci.yml

```diff
@@ -8,21 +8,10 @@ on:
       - master
 
 jobs:
-
   checks:
-
     runs-on: ubuntu-latest
-
     steps:
       - uses: actions/checkout@v1
-        with:
-          submodules: true
-
-      - name: Static checks
-        run: |
-          bash -xc "shellcheck */*.sh"
-          bash -xc "find */ -name '*.sh' | xargs -I {} test -x {}"
-
-      - name: Tests
-        run: |
-          bash -xc "tests/*.sh"
+      - run: make setup
+      - run: make test-static
+      - run: make test-image_push
```

Comment:
we can add the `teardown` here, which is not necessary in Github Actions but helps testing it for regressions?